WORKSPACE

```diff
@@ -61,21 +73,35 @@ go_rules_dependencies()
 
 go_register_toolchains(version = "1.21.5")
 
-load("@io_bazel_rules_docker//repositories:repositories.bzl", container_repositories = "repositories")
-
-container_repositories()
-
-load("@io_bazel_rules_docker//repositories:deps.bzl", container_deps = "deps")
-
-container_deps()
-
-load("@bazel_gazelle//:deps.bzl", "gazelle_dependencies")
-
 gazelle_dependencies()
 
-load("@io_bazel_rules_docker//go:image.bzl", _go_image_repos = "repositories")
-
-_go_image_repos()
+load("@rules_oci//oci:pull.bzl", "oci_pull")
+load("@rules_oci//oci:repositories.bzl", "LATEST_CRANE_VERSION", "oci_register_toolchains")
+
+oci_register_toolchains(
+    name = "oci",
+    crane_version = LATEST_CRANE_VERSION,
+)
+
+# NB: this base image is chosen to match what we had when using rules_docker
+# prior to March 2024:
+# $ bazel query --output=build @go_image_static//image:image
+# container_import(
+#   name = "image",
+#   base_image_digest = "sha256:fac888659ca3eb59f7d5dcb0d62540cc5c53615e2671062b36c815d000da8ef4",
+#   base_image_registry = "gcr.io",
+#   base_image_repository = "distroless/static",
+# )
```

Comment:
Could you remove this comment and the NB below, please? We don't care strongly about using some exact version of the distroless image. I don't see the point in having some arbitrary version of an image documented.